host-interaction/process/terminate/terminate-process.yml

```diff
@@ -23,6 +23,7 @@ rule:
       - api: System.Windows.Forms.Application::Exit
       - api: exit
       - api: Exit
+      - api: exit_group
```

Comment:
please wrap this in an `and` block with an `os: linux` feature, since this isn't relevant on Windows